merge_command.go

```diff
@@ -26,7 +30,9 @@ func handleMergeCommand(issueComment IssueComment, issues Issues, pullRequests P
 		return SuccessResponse{}
 	}
 	err := merge(issueComment.Repository, issueComment.IssueNumber, pullRequests)
-	if err != nil {
+	if err == OutdatedMergeRefError && nrOfRetries > 0 {
```

Comment:
I.e. I still left the variable name as is in the recursive function, but named the constant to be a limit.